src/main/java/mate/academy/springboot/swagger/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.springboot.swagger.dto;
+
+import io.swagger.annotations.ApiModelProperty;
+import java.math.BigDecimal;
+import lombok.AllArgsConstructor;

```

Comment:
Consider adding a NoArgsConstructor annotation for better deserialization support. 😊